### PR TITLE
Add some comment for schemaVersionSyncer

### PR DESCRIPTION
**Pull request by shenli**
schemaVersionSyncer is short of comment.
@zimulala PTAL

### CODE REVIEW

File: ddl/syncer.go [hunk 1]
```diff
@@ -26,24 +26,42 @@ import (
...
+	putKeyRetryInterval   = 30 * time.Millisecond
+	checkVersInterval     = 20 * time.Millisecond
 )
 
 // checkVersFirstWaitTime is used for testing.
 var checkVersFirstWaitTime = 50 * time.Millisecond
 
+// schemaVersionSyncer is used to synchronize schema version between DDL worker leader and follower through etcd.
+//
+// For a leader, during doing DDL job, it should update ddlGlobalSchemaVersion to the latest schema version and wai
```
Comment (zimulala) [inline ddl/syncer.go]: s/wai/wait

**Comment by shenli** [comment]
@zimulala PTAL

**Comment by zimulala** [comment]
LGTM

**Comment by zimulala** [comment]
PTAL @queenypingcap @XuHuaiyu

File: ddl/syncer.go [hunk 2]
```diff
@@ -26,24 +26,42 @@ import (
 )
 
 const (
-	ddlAllSchemaVersions   = "/tidb/ddl/all_schema_versions"
+	// ddlAllSchemaVersions is the path on etcd that every tidb-server instance should register its current loaded schema version.
+	// For a tidb-server instance with a id, it should register its schema version on key: ddlAllSchemaVersions/id. And for each time
+	// the latest schema is loaded, it should update the value of this key.
+	ddlAllSchemaVersions = "/tidb/ddl/all_schema_versions"
+	// ddlGlobalSchemaVersion is the key of the latest schema version of the TiDB cluster.
+	// Its value it the latest schema version. Only DDL worker leader can update its value.
```
Comment (hanfei1991) [inline ddl/syncer.go]: Its value is the .... Only worker leader can update it.

File: ddl/syncer.go [hunk 3]
```diff
@@ -26,24 +26,42 @@ import (
...
 var checkVersFirstWaitTime = 50 * time.Millisecond
 
+// schemaVersionSyncer is used to synchronize schema version between DDL worker leader and follower through etcd.
+//
+// For a leader, during doing DDL job, it should update ddlGlobalSchemaVersion to the latest schema version and wait
+// for all followers to get the latest schema by checking the path ddlAllSchemaVersions on etcd. Then it could move to
+// the next step of the DDL job.
+//
+// For a follower, it should watch ddlGlobalSchemaVersion and reload schema as soon as any change happens. Then it should
+// update it latest schema version to ddlAllSchemaVersions/id on the etcd.
```
Comment (hanfei1991) [inline ddl/syncer.go]: it->its

**Comment by shenli** [comment]
@hanfei1991 PTAL

**Comment by hanfei1991** [comment]
LGTM

**Pull request merged**